composer.json

```diff
@@ -23,7 +23,7 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "phpunit/phpunit": ">=5.7",
```

Comment:
This change allows PHPUnit to be installed for both PHP5.6 and >8

Removing the setup methods below stops the error about return types not matching from different PHPUnit versions

Reply:
Very pog